src/creator.html

```diff
@@ -169,7 +170,26 @@ <h3>Phrase</h3>
 			<span>{{currentPage+1}}/{{numberOfPages()}}</span>
 			<button ng-disabled="currentPage >= numberOfPages() - 1" ng-click="currentPage=currentPage+1">Next</button>
 		</div>
-
+		<div ng-show= "questionBankDialog" class="overlay"></div>
+		<div class="questionBank-wrapper" ng-show= "questionBankDialog">
+				<div class="questionBank-content">
+					<strong>Enable question bank? </strong>
+					<br/>
+					<span>
+						<input type="radio" ng-model="enableQuestionBank" ng-value=false> No</input>
+						<input type="radio" ng-model="enableQuestionBank" ng-value=true> Yes</input>
+					</span>
+					<br/><br/>
+					<span ng-show="enableQuestionBank" >
+						<strong>How many questions to ask?</strong>
+						<br/>
+						<input class="numInput" type="number" ng-model="questionBankVal" ng-attr-min="{{items.length < 1 ? items.length : 1}}" ng-attr-max="{{items.length}}" step="1">
```

Comment:
`ng-attr-min` keeps the user from inputting an integer below 1 here, but the variable being bound to this input is `questionBankVal`, which is initially set to 0. You may want to set the default value of `questionBankVal` to 1 instead, so the input doesn't default to an illegal value.